src/nat/front_ends/fastapi/message_validator.py

```
@@ -258,31 +258,40 @@ async def create_system_response_token_message(
         :return: A WebSocketSystemResponseTokenMessage instance.
         """
         try:
+            # If weave_call_id not provided, try to get from context
+            if weave_call_id is None:
+                try:
+                    from nat.builder.context import Context
+                    weave_call_id = Context.get().weave_call_id
+                except Exception:
+                    pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)